Resource Packs/Mod Menu Helper/assets/fo/lang/en_us.json

```diff
@@ -13,17 +13,17 @@
     "modmenu.summaryTranslation.colormatic": "§6🔀§r OptiFine's Custom Colors\n§3§l📥§r§9🛠§8 Clear sky/void, light smoothing",
     "modmenu.summaryTranslation.continuity": "OptiFine's Connected & Emissive Textures\n§3§l📥§r§9🛠§8 Toggle block features",
     "modmenu.summaryTranslation.controlify": "Game controller support\n§9🛠§8 Vibration, sounds, ...",
-    "modmenu.summaryTranslation.cubes-without-borders": "Full screen with Alt+Tab support (opt-in)\n§9🛠§8 In Video Settings: toggle mod",
+    "modmenu.summaryTranslation.cubes-without-borders": "Full screen with Alt+Tab support (opt-in)\n§9🛠§8 In <lang options.videoTitle>: toggle mod",
     "modmenu.summaryTranslation.cull-less-leaves": "§6🔀§r Faster tree leaves\n§9🛠§8 Toggle mod, leaf depth",
-    "modmenu.summaryTranslation.debugify": "Fixes bugs, disables Mojang's telemetry\n§9🛠§8 See FO wiki for more info",
+    "modmenu.summaryTranslation.debugify": "Fixes bugs, disables Mojang's telemetry\n§9🛠§8 <url 'https://wiki.download.fo/changed-options#fixed-bugs'>See FO wiki for more info</url>",
```

Comment:
These links might change soon, though it is trivial to update them for all languages